File: trunk/java/SR-RBD/src/ddb/restore/RestoreAck.java
package ddb.restore;

import java.io.IOException;

import ddb.communication.DataInputStream;
import ddb.communication.DataOutputStream;
import ddb.msg.MessageType;

public class RestoreAck extends RestoreMessage {

	@Override
	protected void fromBinary(DataInputStream s) throws IOException {
		// empty

	}

	@Override
	protected MessageType getType() {
		return MessageType.RESTORE_ACK;
	}

	@Override
	protected void toBinary(DataOutputStream s) throws IOException {
		// empty
	}

}

File: trunk/C#/RBD/RBD/Restore/Msg/RestoreAck.cs
using RBD.Communication;

namespace RBD.Restore.Msg
{
    public class RestoreAck : RestoreMessage {

        override public void FromBinary(DataInputStream s)
        {
		    // empty

	    }

        override public MessageType GetMessageType()
        {
		    return MessageType.RESTORE_ACK;
	    }

        override public void ToBinary(DataOutputStream s)
        {
		    // empty
	    }

    }
}